server_extensions_api/src/main/java/com/generalbytes/batm/server/coinutil/LnInvoiceData.java

```diff
@@ -0,0 +1,265 @@
+package com.generalbytes.batm.server.coinutil;
+
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+
+public class LnInvoiceData {
```

Comment:
Please add link to the description of this object

Reply:
added